src/typings/types.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Type for Game History variables
+ */
+type GameHistory = Array<{ squares: Array<null | string> }>;
+
+type Action =
```

Comment:
```suggestion
type GameStateAction =
```

I'd be very explicit naming types in the global namespace